### PR TITLE
Fix build

### PR DESCRIPTION
**Pull request by MattiSG**
And decrease published package size.

Oddly, the CSV seems to already not have been included, even while no instruction was given for it.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,6 +19,12 @@
     "url": "https://github.com/sgmap/codes-postaux/issues"
   },
   "homepage": "https://github.com/sgmap/codes-postaux",
+  "files": [
+    "LICENSE.MIT.txt",
+    "codes-postaux.json.gz",
+    "postinstall.js",
+    "index.js"
+  ],
```
Comment (jdesboeufs) [inline package.json]: Duplicates `.npmignore`

Comment (MattiSG) [inline package.json]: Haaaan c'était donc ça.

Comment (MattiSG) [inline package.json]: Which one do we use then?

Comment (MattiSG) [inline package.json]: Blacklist probably decreases risk of bad publishes. I'll revert this set of adds.

**Pull request merged**